crates/serpen/tests/fixtures/ast_rewriting/happy_path/services/database.py

```
@@ -0,0 +1,53 @@
+"""
+Database service module.
+Contains database-related functionality with unique names.
+"""
+
+from typing import Dict, Any, Optional
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)